README.md

```diff
@@ -1 +1,22 @@
-# opentracing-c
\ No newline at end of file
+# opentracing-c
+
+C implementation of the OpenTracing API http://opentracing.io
+
+[![Join the chat at https://gitter.im/opentracing/opentracing-cpp](https://badges.gitter.im/opentracing/opentracing-cpp.svg)](https://gitter.im/opentracing/opentracing-cpp?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+## Required Reading
+
+In order to understand the C platform API, one must first be familiar with the
+[OpenTracing project](http://opentracing.io) and
+[terminology](http://opentracing.io/documentation/pages/spec) more generally.
+
+## C/C++98
+
+This library requires C++11 or later. But if you're interested in a C or C++98
```

Comment:
Actually this would not be the case here. Any C/C++ compiler should be able to do it.

Reply:
okay, I will review it once more.